### PR TITLE
Route the Profile integration + feature flag

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR aims at routing the Profile feature. In this, I

- Routed `/account/profile`
- Added 2 new icons + stories.
- Added a feature flag to prevent the access of the feature on production

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Screenshots

image

image

image

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-h93nhn Create `/account/profile` + fake data fetch](https://app.clickup.com/t/h93nhn)

File: src/pages/account/profile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import { logger } from "@src/configs/logger";
+import getTracer from "@src/configs/tracer";
+import { authMiddleware } from "@src/middlewares/auth-middleware";
+import { sentryMiddleware } from "@src/middlewares/sentry-middleware";
+
+const AccountPage: React.FC = () => {
+  return (
+    <Layout breadcrumbs={false} title="Welcome to your account">
+      <Container>
+        <WIP>🏗</WIP>
```
Comment (Oluane) [inline src/pages/account/profile/index.tsx:25]: 🤩

**Review by Oluane — APPROVED** [review]

File: src/components/pages/account-overview/account-overview.tsx [hunk 2]
```diff
@@ -26,12 +32,17 @@ const AccountOverview: React.FC = () => {
     <>
       {Object.entries(SECTION_LIST_CONTENT).map(
         ([sectionName, { text, icon }]) => {
+          const sectionHref =
+            sectionName.toLocaleLowerCase() === "personal_information"
+              ? "/account/profile"
+              : `/account/${sectionName.toLocaleLowerCase()}`;
+
           return (
```
Comment (Oluane) [inline src/components/pages/account-overview/account-overview.tsx:40]: Need a check for feature flag env var here too, no ?

Comment (Fuzznimp) [reply src/components/pages/account-overview/account-overview.tsx:48]: 74ee1ba

**Review by Oluane — APPROVED** [review]

**Pull request merged**